### ISSUE TITLE
Rides doubling up in new rides window

### ISSUE THREAD

**Issue by chrismilne300** [OpenRCT2/OpenRCT2#3594]
**OS:** windows 10
**Version:** 0.0.5
**Commit:** f62932a

Open the save I've attached and go to build a ride see how some of the rides have double entries.

**Screenshots / Video:**
![screen shot 05-14-16 at 10 23 am 001](https://cloud.githubusercontent.com/assets/1381723/15267610/70df5432-19be-11e6-8a57-f2d2dbbb196b.PNG)
![screen shot 05-14-16 at 10 23 am](https://cloud.githubusercontent.com/assets/1381723/15267611/70df6594-19be-11e6-9286-69965ef3c515.PNG)

**Save game:**
[Save.sv6.txt](https://github.com/OpenRCT2/OpenRCT2/files/264448/Save.sv6.txt)

**Comment by chrismilne300** [comment]
I've investigated this a little more.

Steps to reproduce bug:

1. Make sure you've just opened your game. Start a new game in Crazy Castle
2. Next quite Crazy Castle and start a new game in Electric Fields
3. Open the Object Selection window and close without doing anything else
4. Open the Inventions List window and close again without doing anything else
5. Open the New rides window and notice the double/triple listings.

I think this could also be related to #3562

**Comment by Ziscor** [comment]
It most definitely could be related to #3562 since both things seem to happen together. I edited a Prison Island scenario to make it appropriate for Multiplayer, and the save file I created from that ended up having **Signs and Items for Footpaths** twice. One of them had 3 objects the other didn't, so these duplicates aren't exactly identical; some miss items the other might have.

[Shawshank.sv6.txt](https://github.com/OpenRCT2/OpenRCT2/files/264488/Shawshank.sv6.txt)

Tell me if I should post this instead in #3562 to avoid confusion.
